docs/root/configuration/overview/xds_api.rst

```diff
@@ -348,3 +348,23 @@ To use delta, simply set the api_type field of your
 That works for both xDS and ADS; for ADS, it's the api_type field of
 :ref:`DynamicResources.ads_config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dynamic_resources>`,
 as described in the previous section.
+
+
+.. _config_overview_ttl:
+
+xDS TTL
+^^^^^^^
+
+When using xDS, users might find themself wanting to temporarily update certain xDS resources. In order to do
+safely, xDS TTLs can be used to make sure that if the control plane becomes unavailable and is unable to revert
+the xDS change, Envoy will remove the resource after a TTL specified by the server.
+
+Currently the behavior when a TTL expires is that the resource is *expired* (as opposed to reverted to the
+previous version). As such, this feature should primarily be used for use cases where the absence of the resource
+is preferred instead of the temporary version, e.g. when using RTDS to apply a temporary runtime override.
```

Comment:
I want to add another use case of expiration. For resource collection which need indexed by Envoy(route/virtualhost or filter chains), expiration callback is less expensive.
Expire resource callback could be lea as a place holder. Another on-demand request or find the "most specific unexpired resource" could be provided. These semantic cannot be provided if xds client implicitly revert the resource.

Reply:
Can you elaborate @lambdai with an example?